modules/manage/pages/manage-backup-and-restore/manage-backup-and-restore.adoc

```diff
@@ -15,8 +15,9 @@ This can be configured by means of the *Backup* UI provided by Couchbase Web Con
 * The xref:backup-restore:cbbackupmgr.adoc[cbbackupmgr] CLI utility.
 
 The technology that underlies the Backup Service includes that provided by `cbbackupmgr`: note, however, that `cbbackupmgr` and the Backup Service differ, in terms of the features they offer (as described below, in xref:manage:manage-backup-and-restore/manage-backup-and-restore.adoc#the-backup-service-and-cbbackupmgr[The Backup Service and cbbackupmgr]).
-For use of either the Backup Service or `cbbackupmgr`, the Full Admin role must have been assigned.
-On Couchbase Server 7.0 and above, both the Backup Service and `cbbackupmgr` are available in both the Enterprise and the Community Edition.
+For use of the Backup Service, the Full Admin role must have been assigned.
+For use of `cbbackupmgr`, the Full Admin or the Data Backup & Restore role must have been assigned.
+Both the Backup Service and `cbbackupmgr` are available only in Couchbase Server Enterprise Edition.
```

Comment:
Sorry this may be my fault. From 7.0 onwards cbbackupmgr is available in CE with some limiations in functionality and in EE with all the functionalities.

CE cbbackupmgr **won't** be able to do:
- Merge
- Cloud backup
- Collection level restore
- PITR (I believe this may be DP in 7.0)

The backup service is only available in EE and has all the features.

Reply:
Thanks! I've modified accordingly.